src/ducks/pin/PinGuard.jsx

```diff
@@ -21,9 +21,11 @@ class PinGuard extends React.Component {
   initState() {
     const savedLast = lastInteractionStorage.load()
     const last = savedLast || Date.now()
+    const cachedPinSetting = pinSettingStorage.load()
```

Comment:
Same for `pingSettingStorage`: where does it come from?

Reply:
Problem in the order of commits, solved now.